homeassistant/util/yaml.py

```diff
@@ -119,10 +120,15 @@ def _env_var_yaml(loader, node):
         raise HomeAssistantError(node.value)
 
 
+def _secret_yaml(loader, node):
+    """Load secrets and embed it into the configuration YAML."""
+    return secrets.get_secret(node.value) or ''
```

Comment:
config / secret validation is not part of `load_yaml` so I prefer to blow up for now. I am interested in adding a standalone config validation method later, but in another PR.